src/SMAIAXBackend.API/Endpoints/Order/OrderEndpoints.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace SMAIAXBackend.API.Endpoints.Order;
+
+public static class OrderEndpoints
+{
+    public static WebApplication MapOrderEndpoints(this WebApplication app)
+    {
+        const string contentType = "application/json";
+        var group = app.MapGroup("/api/orders")
+            .WithTags("Order")
+            .RequireAuthorization();
+
+        group.MapPost("/", OrderSmartMeterConnectorEndpoint.Handle)
+            .WithName("OrderSmartMeterConnector")
+            .Accepts<object>(contentType)
```

Comment:
```suggestion
```
Remove this line. The request body not used.
![image](https://github.com/user-attachments/assets/a5a84ab9-226a-4e87-b067-7ed7f8ea7fec)

Without is better:
![image](https://github.com/user-attachments/assets/c63103f7-28d4-4fe1-8e27-ba3b93cf3b43)